mysql/Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM localhost:5000/ubuntu
 
-RUN mkdir /etc/service/mysql && \
+RUN mkdir -p /etc/service/mysql && \
 
     apt-get update && \
     apt-get -y --no-install-recommends install mysql-server && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 COPY my.cnf /etc/mysql/my.cnf
+COPY mysql.mk /usr/share/tugboat/mysql.mk
 
-RUN /usr/sbin/mysqld & \
+RUN find /var/lib/mysql -type f -exec touch {} \; && \
```

Comment:
@bchavet this is what I had to add to get this image to build on my local.

Reply:
The image builds fine on production with this, too. +1 to leave it here